qt/widgets/common/CMakeLists.txt

```diff
@@ -94,74 +269,75 @@ set ( SRC_FILES
   src/Batch/QtTreeCursorNavigation.cpp
   src/Batch/CellDelegate.cpp
   src/Batch/BuildSubtreeItems.cpp
-	src/DataSelector.cpp
-	src/DiagResults.cpp
-	src/DoubleSpinBox.cpp
-	src/FindDialog.cpp
-        src/FindFilesThreadPoolManager.cpp
-        src/FindFilesWorker.cpp
-	src/FindReplaceDialog.cpp
-	src/FitOptionsBrowser.cpp
-	src/FitPropertyBrowser.cpp
-	src/FunctionBrowser.cpp
-	src/HintingLineEdit.cpp
-	src/IndirectFitPropertyBrowser.cpp
-	src/IndirectInstrumentConfig.cpp
-	src/InputController.cpp
-	src/InstrumentSelector.cpp
-	src/LineEditWithClear.cpp
-	src/LogValueSelector.cpp
-	src/MWDiag.cpp
-	src/MantidHelpWindow.cpp
-	src/MantidTreeModel.cpp
-	src/MantidTreeWidget.cpp
-	src/MantidTreeWidgetItem.cpp
-	src/MantidWSIndexDialog.cpp
-	src/MessageDisplay.cpp
-	src/MultifitSetupDialog.cpp
-	src/MuonFitDataSelector.cpp
-	src/MuonFitPropertyBrowser.cpp
-	src/MuonFunctionBrowser.cpp
-	src/PeriodicTableWidget.cpp
-	src/ProcessingAlgoWidget.cpp
-        src/ProgressableView.cpp
+  src/DataSelector.cpp
+  src/DiagResults.cpp
+  src/DoubleSpinBox.cpp
+  src/FindDialog.cpp
+  src/FindFilesThreadPoolManager.cpp
+  src/FindFilesWorker.cpp
+  src/FindReplaceDialog.cpp
+  src/FitOptionsBrowser.cpp
+  src/FitPropertyBrowser.cpp
+  src/FunctionBrowser.cpp
+  src/HintingLineEdit.cpp
+  src/IndirectFitPropertyBrowser.cpp
+  src/IndirectInstrumentConfig.cpp
+  src/InputController.cpp
+  src/InstrumentSelector.cpp
+  src/LineEditWithClear.cpp
+  src/LogValueSelector.cpp
+  src/MWDiag.cpp
+  src/MantidHelpWindow.cpp
+  src/MantidTreeModel.cpp
+  src/MantidTreeWidget.cpp
+  src/MantidTreeWidgetItem.cpp
+  src/MantidWSIndexDialog.cpp
+  src/MessageDisplay.cpp
+  src/MultifitSetupDialog.cpp
+  src/MuonFitDataSelector.cpp
+  src/MuonFitPropertyBrowser.cpp
+  src/MuonFunctionBrowser.cpp
+  # todo check if it can be removed from qt4
```

Comment:
Can we remove the todo comments, please?

Reply:
Comments have been removed 👍